docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -248,6 +248,13 @@ You can read more about transaction isolation and snapshot features at
    GRANT ALL PRIVILEGES ON DATABASE airflow TO airflow_user;
 
 
+We recommend using the ``mssql+pyodbc`` driver and specifying it in your SqlAlchemy connection string.
+
+.. code-block:: text
+
+    mssql+pyodbc://<user>:<password>@<host>
```

Comment:
On CI, we use a little more complex URI. This leads me to suppose that this example may be incomplete. https://github.com/apache/airflow/blob/main/scripts/ci/docker-compose/backend-mssql.yml#L23-L24

Reply:
This is the default one from the sqlalchemy docs:

https://docs.sqlalchemy.org/en/14/core/engines.html#microsoft-sql-server 

We use mor complex one, but those are optional parameters, and I wouldn't dive into details of it (especially not before we have first users using 'for real' and making comments about it.

Reply:
Should we add a database name to the example? I think it is a commonly used parameter. It needed to set up a database created in the paragraph above. 

```
mssql+pyodbc://<user>:<password>@<host>
mssql+pyodbc://<user>:<password>@<host>/<database_name>
```

Reply:
I created a PR: https://github.com/apache/airflow/pull/18404